Sources/SwiftSDKGenerator/Generator/SwiftSDKGenerator.swift

```diff
@@ -136,43 +136,12 @@ public actor SwiftSDKGenerator {
     #endif
   }
 
-  private func buildDockerImage(name: String, dockerfileDirectory: FilePath) async throws {
```

Comment:
Short to medium term, we should bring back the ability to generate from user-provided or user-customized Dockerfiles, as we did in the initial implementation. We even have (also vestigial) example Dockerfile for it: https://github.com/apple/swift-sdk-generator/blob/49986df3e3c9f6d8cbee03cb8cba37897aec98d8/Sources/SwiftSDKGenerator/Dockerfiles/Ubuntu/22.04/Dockerfile#L5

It became unused after one of the later refactorings. Your change is fine as technically nothing breaks, but I think we should keep these `buildDockerImage` functions even as unused, with a `FIXME` comment reminding us to restore the ability for users to customize build images.

Reply:
To elaborate, not calling `docker build` is the correct way to do it right now, but let's not remove `private func buildDockerImage` just yet, in anticipation of future changes that make use of it.